kubernetes/structures_container.go

```diff
@@ -348,15 +348,11 @@ func flattenContainerPorts(in []v1.ContainerPort) []interface{} {
 	return att
 }
 
-func flattenContainerResourceRequirements(in v1.ResourceRequirements) ([]interface{}, error) {
```

Comment:
Remove the `error` return from this function since we never return any errors, just `nil`.